inc/database/storage-handler.php

```diff
@@ -85,7 +85,9 @@ public function delete_object_data( $object_id ) {
 		$object_type   = str_replace( array( 'deleted_', 'delete_' ), '', current_filter() );
 		$relationships = $this->factory->filter_by( $object_type );
 		foreach ( $relationships as $relationship ) {
-			$this->delete_object_relationships( $object_id, $relationship->id );
+			$setting = $this->factory->get_settings( $relationship->id );
+			$target  = $setting['from']['object_type'] === $object_type ? 'from' : 'to';
+			$this->delete_object_relationships( $object_id, $relationship->id, $target );
```

Comment:
Anh nghĩ trong trường hợp mà object type ở 2 phía `from` và `to` khác nhau thì code này chạy chuẩn.

Nhưng nếu mình xử lý trường hợp object type ở 2 phía giống nhau (VD từ post tới page) thì như thế này mình chỉ xoá được 1 chiều của relationship, vẫn còn sót trong DB. Trường hợp này code cũ thì xử lý đúng. Em xem fix trường hợp này nhé.